s3_bucket.go

```diff
@@ -1323,3 +1325,64 @@ func (s *s3ArchiveBucket) Pull(ctx context.Context, opts SyncOptions) error {
 
 	return nil
 }
+
+// PresignExpireTime sets the amount of time the link is live before expiring.
+const PresignExpireTime = 24 * time.Hour
```

Comment:
Why not make this configurable?

Reply:
Since I'm only moving this from evergreen to pail, it would be out of scope for this ticket.